project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -224,6 +224,28 @@ function c4m_og_menu() {
 function c4m_og_menu_alter(&$items) {
   // Adding a member should be allow to SA only (CFM-1292).
   $items['group/%/%/admin/people/add-user']['access callback'] = 'c4m_og_add_member_to_group_access';
+
+  // Assign access to Node Gallery API features, if user has permission to
+  // edit the gallery.
+  $node_galery_api_features = array('upload', 'items', 'manage', 'sort');
+  foreach ($node_galery_api_features as $suffix) {
+    $path = 'node/%node_gallery_api_gallery/' . $suffix;
+    $items[$path]['access callback'] = 'node_access';
+    $items[$path]['access arguments'] = array('update', 1);
+  }
+
+  // Assign access to upload Node Gallery API item types, per upload access
+  // function.
+  // Node Gallery API item types are resolve in same manner
+  // as at node_gallery_api_menu().
+  $relationship_types = node_gallery_api_get_relationship_type(NULL, NULL, NULL, TRUE);
+  foreach ($relationship_types as $relationship) {
+    foreach ($relationship->item_types as $item_type) {
+      $path = 'node/%node_gallery_api_gallery/upload/' . $item_type;
```

Comment:
let's drop the `$path` and use it directly in the `$items` variable (more readable for `hook_menu_alter`).